### PR TITLE
reinstate etcdctl

### PR DESCRIPTION
**Pull request by pbx0**
https://github.com/coreos/mantle/issues/131

### CODE REVIEW

File: kola/tests/etcd/util.go [hunk 1]
```diff
@@ -24,43 +24,38 @@ import (
...
-    discovery: $discovery
-    advertise-client-urls: http://$private_ipv4:2379
-    initial-advertise-peer-urls: http://$private_ipv4:2380
-    listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
-    listen-peer-urls: http://$private_ipv4:2380,http://$private_ipv4:7001`,
-	})
+// GetClusterHealth polls etcdctl cluster-health command until success
+// or maximum retries have been reached. Can be effectively used to
+// block a test until the etcd cluster is up and running.
+func GetClusterHealth(m platform.Machine, csize int) error {
```
Comment (mischief) [inline kola/tests/etcd/util.go]: any reason to export this?

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

File: kola/tests/etcd/util.go (hunk 1 above)
Comment (pbx0) [inline kola/tests/etcd/util.go]: Its used it kola/tests/coretest/etcd.go to wait for etcd to come up.

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
lgtm

**Pull request merged**